moped-editor/src/views/projects/projectView/ProjectComponentsMapView.js

```diff
@@ -78,6 +79,10 @@ const ProjectComponentsMapView = ({
    * Let's throw an error intentionally if there are no features for a project.
    */
   if (featureCount < 1) {
+    // If this is local, disable react error overlay
+    stopReportingRuntimeErrors();
+
+    // Now throw the error the error boundary can catch
```

Comment:
Gets rid of the annoying error message whenever we open a project without components.